IsoCountries.swift

```diff
@@ -15,10 +15,22 @@ struct IsoCountryInfo {
     var calling:String
     var currency:String
     var continent:String
+    var flag: String {
+        return IsoCountries.flag(countryCode: alpha2)
+    }
 }
 
 class IsoCountries {
     
+    class func flag(countryCode: String) -> String {
+        var string = ""
+        var country = countryCode.uppercased()
+        for uS in country.unicodeScalars {
+            string.unicodeScalars.append(UnicodeScalar(127397 + uS.value)!)
```

Comment:
• Can the `UnicodeScalar(127397 + uS.value)!` be guarded before appending? That way you are sure the scalar value exists and you can safely add it. This _could_ potentially crash when force unwrapping a `nil` value.

• Also what is the `127397` magic number? (Can you explain?)